app/scripts/views/mixins/password-reset-mixin.js

```diff
@@ -24,18 +24,17 @@ define(function (require, exports, module) {
      * @return {promise} - resolves with auth server response if successful.
      */
     resetPassword: function (email) {
-      var self = this;
-
       var account = this.user.initAccount({ email: email });
       return account.resetPassword(
-        self.relier,
+        this.relier,
         {
-          resume: self.getStringifiedResumeToken()
```

Comment:
Whenever I run this manually and step into `getStringifiedResumeToken`, `this.flow` is always `undefined`. I think you either need to use the `flow` model passed from the signin view (if passed) or create one locally?